velox/common/base/tests/StatsReporterTest.cpp

```diff
@@ -92,14 +92,14 @@ TEST_F(StatsReporterTest, trivialReporter) {
   auto reporter = std::dynamic_pointer_cast<TestReporter>(
       folly::Singleton<BaseStatsReporter>::try_get());
 
-  REPORT_ADD_STAT_EXPORT_TYPE("key1", StatType::COUNT);
-  REPORT_ADD_STAT_EXPORT_TYPE("key2", StatType::SUM);
-  REPORT_ADD_STAT_EXPORT_TYPE("key3", StatType::RATE);
+  REPORT_ADD_STAT_EXPORT_TYPE("key1", StatType::kCount);
+  REPORT_ADD_STAT_EXPORT_TYPE("key2", StatType::kSum);
+  REPORT_ADD_STAT_EXPORT_TYPE("key3", StatType::kRate);
   REPORT_ADD_HISTOGRAM_EXPORT_PERCENTILE("key4", 10, 0, 100, 50, 99, 100);
 
-  EXPECT_EQ(StatType::COUNT, reporter->statTypeMap["key1"]);
-  EXPECT_EQ(StatType::SUM, reporter->statTypeMap["key2"]);
-  EXPECT_EQ(StatType::RATE, reporter->statTypeMap["key3"]);
+  EXPECT_EQ(StatType::kCount, reporter->statTypeMap["key1"]);
```

Comment:
This refactoring is nice, but it makes it hard to review the material changes. Next time, consider separating refactoring changes into their own PR.